aws/rust-runtime/aws-types/src/build_metadata.rs

```diff
@@ -41,7 +41,7 @@ pub enum OsFamily {
 /// is set to a specific value. This macro simplifies checking the current OS family.
 ///
 /// Usage:
-/// ```rust
+/// ```ignore
```

Comment:
See https://github.com/awslabs/smithy-rs/commit/e68dfb7be743f1b4320deb33028424c154d8ce44 for the reason on the `ignore` here.

`rust` is redundant and can be removed.